File: Signiture-Backend/delivery/api/src/main/java/pcy/study/api/domain/user/controller/model/UserLoginRequest.java
package pcy.study.api.domain.user.controller.model;

import io.swagger.v3.oas.annotations.media.Schema;
import jakarta.validation.constraints.NotBlank;

public record UserLoginRequest(
        @Schema(description = "이메일", example = "example@gmail.com")
        @NotBlank
        String email,
        @Schema(description = "비밀번호", example = "pws123")
        @NotBlank
        String password
) {
}

File: Signiture-Backend/kotlin/delivery/api/src/main/kotlin/pcy/study/api/domain/user/controller/model/UserLoginRequest.kt
package pcy.study.api.domain.user.controller.model

import io.swagger.v3.oas.annotations.media.Schema
import jakarta.validation.constraints.NotBlank

data class UserLoginRequest(
    @field:Schema(description = "이메일", example = "example@gmail.com")
    val email: @NotBlank String,
    @field:Schema(description = "비밀번호", example = "pws123")
    val password: @NotBlank String,
)